scripts/globals/items/teleport_ring_yhoat.lua

```diff
@@ -1 +1 @@
------------------------------------------
--    ID: 14665
--    Teleport ring: Yhoat
--  Enchantment: "Teleport-Yhoat"
-----------------------------------------

require("scripts/globals/status");
require("scripts/globals/keyitems");
require("scripts/globals/teleports");

-----------------------------------------
-- OnItemCheck
-----------------------------------------

function onItemCheck(target)
    local result = 0;
    if (target:hasKeyItem(YHOATOR_GATE_CRYSTAL) == false) then
        result = 445;
    end
    return result;
end;

-----------------------------------------
-- OnItemUse
-----------------------------------------

function onItemUse(target)
    target:addStatusEffectEx(EFFECT_TELEPORT,0,TELEPORT_YHOAT,0,0.5);
end;
\ No newline at end of file
+-----------------------------------------
--    ID: 14665
--    Teleport ring: Yhoat
--  Enchantment: "Teleport-Yhoat"
-----------------------------------------

require("scripts/globals/status");
require("scripts/globals/keyitems");
require("scripts/globals/teleports");

-----------------------------------------
-- OnItemCheck
-----------------------------------------

function onItemCheck(target)
    local result = 0;
    if (target:hasKeyItem(YHOATOR_GATE_CRYSTAL) == false) then
        result = 445;
    end
    return result;
end;

-----------------------------------------
-- OnItemUse
-----------------------------------------

function onItemUse(target)
    target:addStatusEffectEx(EFFECT_TELEPORT,0,TELEPORT_YHOAT,0,1);
end;
```

Comment:
Odd github normally doesn't show those like that. Or didn't. They must have altered something in their backend recently. We're supposed to have unix line ends on our script files anyway so its not a aproblem.